tasks.yaml

```diff
@@ -125,7 +126,8 @@ tasks:
     description: "Integration Tests"
     actions:
       - task: cypress-tests
-      - task: theme-customization-tests
+#      - task: theme-customization-tests
```

Comment:
Should this be re-added somewhere (or is it already) to make sure we don't lose coverage?